### ISSUE TITLE
Show "See Also" links for all functions

### ISSUE THREAD

**Issue by cskksc** [clojure-emacs/cider#1746]
Right now (#1736), the middleware has `See Also` information only for `clojure.core`.
But it would be good to be able to show the links for all namespaces. Parsing the docstring and generating links from there is one way to do it. So, the docstring  would look something like,

```
Returns a lazy sequence of the items in coll for which
(pred item) returns true. pred must be free of side-effects.
Returns a transducer when no collection is provided.

See also: `clojure.core/remove', `clojure.core/keep', `user-ns/user-fn'
```

So we just need to collect all the `` `' `` pairs and replace them with buttons which would lookup the corresponding documentation.

Relevant discussion [here](https://github.com/clojure-emacs/cider/issues/1736#issuecomment-217754434)

**Comment by Malabarba** [comment]
Two things:

1. We should use ``` `clojure.core/remove` ```, not ``` `clojure.core/remove'```.
2. We should try to linkify any occurrence of ``` `function-in-quotes` ``` in the docstring, not just the "see also:" links.

**Comment by cskksc** [comment]
Fixed by #1750.
